modules/installation-configuration-parameters.adoc

```diff
@@ -462,11 +460,14 @@ ifndef::aws,bare,gcp,ibm-power,ibm-z,azure,ibm-power-vs[]
 |String
 endif::aws,bare,gcp,ibm-power,ibm-z,azure,ibm-power-vs[]
 
-ifdef::aws,bare,azure,gcp[]
+ifdef::aws,azure,gcp,bare[]
 |`compute.architecture`
-|Determines the instruction set architecture of the machines in the pool. Currently, clusters with varied architectures are not supported. All pools must specify the same architecture. Valid values are `amd64` and `arm64`. See _Supported installation methods for different platforms_ in _Installing_ documentation for information about instance availability.
+|Determines the instruction set architecture of the machines in the pool. Currently, clusters with varied architectures are not supported. All pools must specify the same architecture. Valid values are `amd64` and `arm64`. 
 |String
-endif::aws,bare,azure,gcp[]
+ifdef::aws,azure[]
```

Comment:
I believe the condition is in the wrong spot. It needs to be included in the column defined by line 465.